lib/package.json

```diff
@@ -24,7 +24,9 @@
     "@vanilla-extract/sprinkles": "^1.3.3",
     "@vanilla-extract/vite-plugin": "^3.1.0",
     "deepmerge": "^4.2.2",
-    "polished": "^4.1.3"
+    "polished": "^4.1.3",
+    "react": "^17.0.0",
+    "react-dom": "^17.0.0"
```

Comment:
Can we make this, above or equal to 17?
```suggestion
    "react": ">=17.0.0",
    "react-dom": ">=17.0.0"
```

Ref:
https://nodejs.org/en/blog/npm/peer-dependencies/
<img width="1016" alt="image" src="https://user-images.githubusercontent.com/30571073/184508111-4ef5f661-c765-46a3-8830-9a855e838c3a.png">

Reply:
sure we can do this, as of today, i do not see any issue